docs/API_POLICY.md

```diff
@@ -0,0 +1,64 @@
+# Hub API Policy
+
+This document proposes a policy regarding making updates to the APIs of this repo.
+
+NOTE: Hub Release Version and API version are different and may not be in sync. Making a Hub major release does not necessarily increment API version too. API version will be incremented only when there is a breaking change.
+eg. `/v1` -> `/v2` only when there is a breaking change.
+
+Hub API service hosts two types of APIs:
+
+## 1. Versioned APIs
+
+- These are the APIs which are exposed for external users to use and integrate with their applications.
+- These will have support provided by the Hub team.
+- These are the APIs which return hub resource data.
+- The version of the APIs will be prefixed to the API route. 
+  Eg. resource API - /v1/resources
+- Swagger Documentation for these APIs can be found at `https://api.hub.tekton.dev/<version>/schema/swagger.json`. Replace `<version>` with current version of APIs.
+ 
+
+#### Additive changes
+
+- Additive changes are changes that are added to the APIs and do not cause problems for existing users of the API.
```

Comment:
Additive changes are changes that are added to the APIs in a backward compatible way and thus, do not cause problems for existing users of the API.